openraft/src/core/append_entries.rs

```diff
@@ -204,15 +205,17 @@ impl<D: AppData, R: AppDataResponse, N: RaftNetwork<D>, S: RaftStorage<D, R>> Ra
             return Ok(());
         }
 
-        if msg_entries[0].log_id.index > self.last_log_id.index {
-            return Ok(());
+        if let Some(last_log_id) = self.last_log_id {
+            if msg_entries[0].log_id.index > last_log_id.index {
+                return Ok(());
+            }
         }
 
         tracing::debug!(
             "delete inconsistent log entries [{}, {}), last_log_id: {}, entries: {}",
             msg_entries[0].log_id,
             msg_entries[l - 1].log_id,
-            self.last_log_id,
+            self.last_log_id.map_or_else(|| "None".to_string(), |log_id| log_id.to_string()),
```

Comment:
replace `last_log_id: {}` with `last_log_id: {:?}` to print an `Option<LogId>` directly.